.travis.yml

```diff
@@ -32,7 +45,7 @@ install:
 - pip install wheel pytest pytest-cov absl-py --upgrade
 - if [ "$TF" = "2" ]; then pip install tensorflow==2.1; fi
 - if [ "$TF" = "1" ]; then pip install tensorflow==1.13.2; fi
```

Comment:
Even if the existing 1.X code doesn't give us problems, I worry it will be confusing to users that some parts of the code base work with both, and some don't.

For the low-level non-Keras interface, it's quite easy to support both. We just need to make sure that conditionals or try-excepts are used to import eager-specific modules from the right place (import locations changes between 1.X and 2.X).

For the `qnn` module and Keras related code, however, I'm a lot more wary. Unless the `qnn` tests pass on TF 1.X, safer to assume it is only TF 2.0 compatible.

Reply:
I just tried, and it looks nontrivial to get the Keras layers to work with 1.X.

I think we could get it to work but it would need some changes.

Reply:
Definitely not a priority, I think sunsetting 1.X support is much more likely than extending the qnn module to work with 1.X.